lib/hydra/derivatives/image.rb

```diff
@@ -33,7 +34,7 @@ def new_mime_type(format)
 
       def create_resized_image(output_file, size, format, quality=nil)
         create_image(output_file, format, quality) do |xfrm|
-          xfrm.resize(size) if size.present?
+          xfrm.scale(size) if size.present?
```

Comment:
Benchmarking scale versus resize it turns out the former is a bit faster (see http://www.imagemagick.org/Usage/resize/ for more technical details). For thumbnail sized images the difference in quality may not matter. I could also be just as convinced to change it back to resize since it was just a slight performance tweak.